client/src/Components/Layouts/Captains/style.css

```diff
@@ -88,4 +142,70 @@
 }
 .add-captain__form .ant-modal{
   margin:0 19%;
-} 
\ No newline at end of file
+} 
+
+.ant-calendar-footer-btn {
+  display: none;
+}
+
+.ant-calendar {
+  direction: rtl !important;
+}
+
+.ant-calendar-picker {
+  direction: rtl;
+  border-radius: 2px !important;
+}
+
+.ant-calendar-body {
```

Comment:
@Jamalat-shamallakh If you have an .ant-calendar inside another component, this style will affect that component too.  If you only want it to affect the calendar inside this Captains component, you'll need to make this selector more specific (for example `.containercustomers .ant-calendar-body` and so on).